torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -319,6 +322,15 @@ def callback(fut):
 
 
 class RpcTest(RpcAgentTestFixture):
+    def _skip_if_tensorpipe_agent(old_func):  # noqa
```

Comment:
nit: do we have a type for this noqa? e.g. F401, etc.

Reply:
Yes, it's B902 - just added the types to the noqa